src/libecalc/fixtures/cases/minimal/minimal_yaml.py

```diff
@@ -37,3 +37,34 @@ def minimal_model_yaml(fuel_rate: Union[int, str] = 50):
         )
 
     return minimal_model_yaml
+
+
+@pytest.fixture
+def minimal_model_yaml_factory_2():
```

Comment:
reuse the model above but add extra parameters for fuel and installation name (with default values as before)? To easier spot and know the difference between the models?